data-types.md

```diff
@@ -237,6 +243,9 @@ A Trip is defined by the following structure:
 | `standard_cost`          | Integer         | Optional | The cost, in the currency defined in `currency`, to perform that trip in the standard operation of the System (see [Costs & Currencies][costs-and-currencies]) |
 | `actual_cost`            | Integer         | Optional | The actual cost, in the currency defined in `currency`, paid by the customer of the *mobility as a service* provider (see [Costs & Currencies][costs-and-currencies]) |
 | `currency`               | String          | Optional, USD cents is implied if null.| An [ISO 4217 Alphabetic Currency Code][iso4217] representing the currency of the payee (see [Costs & Currencies][costs-and-currencies]) |
+| `stop_sequence` | Integer | Required if Applicable | Order of stops for a particular trip. The values must increase along the trip, but do not need to be consecutive. |
+| `route_id` | UUID | Required if Applicable | A unique route ID for the associated route and stops. Matches [GTFS](https://gtfs.org/documentation/schedule/reference/) `route_id` |
+| `service_id` | UUID | Required if Applicable | A unique service ID for the set of dates when service is available for a route. Matches [GTFS](https://gtfs.org/documentation/schedule/reference/) `service_id` |
```

Comment:
Just making a note here for the 'Files changed' view that these fields are for **Trips**.